src/components/quicklaunch.jsx

```diff
@@ -219,7 +273,15 @@ export default function QuickLaunch({
                           </div>
                         )}
                         <div className="flex flex-col md:flex-row text-left items-baseline mr-4 pointer-events-none">
-                          <span className="mr-4">{r.name}</span>
+                          {r.type !== "searchSuggestion" && <span className="mr-4">{r.name}</span>}
+                          {r.type === "searchSuggestion" && (
+                            <>
+                              <span className="whitespace-pre">{searchString}</span>
+                              <span className="mr-4 whitespace-pre opacity-50">
+                                {r.name.substring(searchString.length)}
```

Comment:
```suggestion
                              <span className="whitespace-pre">{r.name.indexOf(searchString) === 0 ? searchString : ''}</span>
                              <span className="mr-4 whitespace-pre opacity-50">
                                {r.name.indexOf(searchString) === 0 ? r.name.substring(searchString.length) : r.name}
```

I think something simple like this would do it, but realize you may be working on it

<img width="686" alt="Screenshot 2024-01-30 at 11 18 09 AM" src="https://github.com/gethomepage/homepage/assets/4887959/f7b6b0c9-2e81-463b-8e27-c0ab2e442646">
<img width="676" alt="Screenshot 2024-01-30 at 11 18 19 AM" src="https://github.com/gethomepage/homepage/assets/4887959/93a7b37c-1f17-42ca-b6de-57578d905615">

Reply:
Just playing with them a bit aren't the suggestions always going to be at the beginning of the string if its in the suggestion at all?

Reply:
No, unfortunately not.

 See the `http://gmx.at`

Reply:
Okay, I gave up on trying my idea...

I would probably implement your first idea because with your second one, if you spell the search wrong, everything full opacity. I think it looks better all with reduced opacity in that case, because nothing is actually matching the input.

![image](https://github.com/gethomepage/homepage/assets/31217036/7230b0e0-ad2b-43b2-b7f6-33fd32e1c973)

![image](https://github.com/gethomepage/homepage/assets/31217036/57143a6f-cc9f-4bd5-bdd1-ed6db14cc774)

Reply:
Ok yea Im good with that.